app/uk/gov/hmrc/customs/notification/services/PushClientNotificationService.scala

```diff
@@ -32,14 +33,21 @@ import scala.concurrent.duration.Duration
 @Singleton
 class PushClientNotificationService @Inject() (pushNotificationServiceConnector: PushNotificationServiceConnector,
                                                gaConnector: GoogleAnalyticsSenderConnector,
-                                               notificationLogger: NotificationLogger) {
+                                               notificationLogger: NotificationLogger,
+                                               metricsConnector: CustomsNotificationMetricsConnector,
+                                               dateTimeService: DateTimeService) {
 
   private implicit val hc = HeaderCarrier()
 
   def send(declarantCallbackData: DeclarantCallbackData, clientNotification: ClientNotification): Boolean = {
 
     val pushNotificationRequest = pushNotificationRequestFrom(declarantCallbackData, clientNotification)
 
+    clientNotification.metricsStartDateTime.fold() { startTime =>
+      metricsConnector.post(CustomsNotificationsMetricsRequest(
+        "NOTIFICATION", clientNotification.notification.conversationId,DateTimeUtils.convertDateTimeToZonedDateTime(startTime), dateTimeService.zonedDateTimeUtc))
```

Comment:
`,DateTimeUtils` should be `, DateTimeUtils`

Reply:
ok will fix